src/poliastro/twobody/perturbations.py

```diff
@@ -0,0 +1,38 @@
+from astropy import units as u
+import numpy as np
+from poliastro.util import norm
+from poliastro.bodies import Sun, Earth
+
+
+def J2_perturbation(t0, state, k, J2, R):
+    r"""Calculates J2_perturbation acceleration (km/s2)
+
+    Parameters
+    ----------
+    t0 : float
+        Current time (s)
+    state : numpy.ndarray
+        Six component state vector [x, y, z, vx, vy, vz] (km, km/s).
+    k : float
+        gravitational constant, (km^3/s^2)
+    J2: float
+        obliteness factor
+    R: float
+        attractor radius
+
+    Notes
+    -----
+    The J2 accounts for the obliteness of the attractor. The formula is given in
+    Howard Curtis, (12.30)
+
+    """
+
+    r_vec = state[:3]
+    r = norm(r_vec)
+
+    factor = (3.0 / 2.0) * k * J2 * (R.to(u.km).value ** 2) / (r ** 5)
```

Comment:
I think we should keep the perturbations units-agnostic, at least for now. The API would look something like this:

```
r, v = cowell(orbit, tof, ad=J2_perturbation, J2=Earth.J2.value, R=Earth.R.to(u.km).value)
```

I prefer to start with this (ugly) API and then improve when we figure out than having to regret adding it and go back because the performance is bad.